plugins/seagate/seagate-nvme.c

```diff
@@ -712,15 +732,320 @@ static void json_print_smart_log_CF(struct json_object *root,
 	*/
 }
 
+
+static void print_stx_smart_log_C0(STX_EXT_SMART_LOG_PAGE_C0 *pLogPageC0)
+{
+	
+	printf("\n\nSeagate SMART Health Attributes :\n");
+	printf("%-39s %-19s \n", "Description", "Health Attributes");
+
+	printf("%-40s", "Physical Media Units Written");
+	printf(" 0x%016"PRIx64"%016"PRIx64"", le64_to_cpu(pLogPageC0->phyMediaUnitsWrt.MS__u64),
+	       le64_to_cpu(pLogPageC0->phyMediaUnitsWrt.LS__u64));
+	printf("\n");
```

Comment:
Avoid the additional printf("\n"); by adding the new line to the previous printf?